src/families/bitcoin/wallet-btc/explorer/index.ts

```diff
@@ -54,7 +54,15 @@ class BitcoinLikeExplorer extends EventEmitter implements IExplorer {
     const client = axios.create(clientParams);
     this.underlyingClient = client;
     // 3 retries per request
-    axiosRetry(client, { retries: 3 });
+    axiosRetry(client, {
+      retries: 3,
+      retryCondition: (e) =>
+        isNetworkOrIdempotentRequestError(e) ||
+        // workaround for explorers v3 that sometimes returns 4xx instead of 5xx
+        (e.code !== "ECONNABORTED" &&
+          (!e.response ||
+            (e.response.status >= 400 && e.response.status <= 499))),
```

Comment:
Are 5xx errors still covered?

Reply:
yes. it's part of isNetworkOrIdempotentRequestError